src/cursor.rs

```diff
@@ -72,6 +72,66 @@ impl Command for MoveTo {
     }
 }
 
+/// A command that moves the terminal cursor up the given number of lines, and resets it to the first column at the beginning of the line.
+///
+/// # Notes
+///
+/// Commands must be executed/queued for execution otherwise they do nothing.
+struct MoveToNextLine(u16);
+
+impl Command for MoveToNextLine {
+    type AnsiType = String;
+
+    fn ansi_code(&self) -> Self::AnsiType {
+        ansi::move_to_next_line_csi_sequence(self.0)
+    }
+
+    #[cfg(windows)]
+    fn execute_winapi(&self) -> Result<()> {
+        sys::move_to_next_line(self.0)
+    }
+}
+
+/// A command that moves the terminal cursor down the given number of lines, and resets it to the first column at the beginning of the line.
```

Comment:
* Contributing guidelines violation - line is too long
* _resets it to the first column at the beginning of the line_ - sounds like the same thing is said twice. What about just `and moves it to the first column.`?